### PR TITLE
Create _discord.rrcoder0167.json

### PR DESCRIPTION
**Pull request by riddhimanrana**
This is just a pr for adding my domain to discord connections. I already have a real website at [https://rrcoder0167.is-a.dev](url), I just need to add a discord connection

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

[https://_discord.rrcoder0167.json](url)

### CODE REVIEW

**Comment by rrcoder0167** [comment]
Hi mods!
I also added this code.rrcoder0167.is-a.dev, and thats for a different purpose. Essentially there's this service called [code-server/server](https://coder.com/docs/code-server/latest) that let's you run VScode in the browser from anywhere, and so I was setting up an A record to link it to my machine so I could run VScode anywhere. It seems to have gone into the same PR so I apologize for that. The _discord file, that's just me for verifying my discord connections and auth.

Thank you for your understanding!

File: domains/code.rrcoder0167.is-a.dev
Comment (Stef-00012) [inline domains/code.rrcoder0167.is-a.dev:1]: file should be a JSON file

File: domains/code.rrcoder0167.is-a.dev [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+       "username": "rrcoder0167",
+       "email": "riddhiman.rana@gmail.com",
+       "discord": "870936028108705803"
+    },
+
+    "record": {
+        "A": "192.168.86.22"
```
Comment (Stef-00012) [inline domains/code.rrcoder0167.is-a.dev:9]: A record should be an array of IPs

Comment (Stef-00012) [inline domains/code.rrcoder0167.is-a.dev:9]: also can not reach your website, keeps on loading

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by orxngc** [comment]
Stale

**Pull request closed without merging**